randovania/games/game.py

```diff
@@ -171,6 +170,9 @@ class GameData:
     web_info: GameWebInfo = GameWebInfo()
     """Contains a handful of fields displayed primarily on the website."""
 
+    hash_words: list[str] | None = None
```

Comment:
They way you ended up doing, hash words is not None for all our games and new games end up copying Blank's word list. So you can make this required.

Reply:
Yeah, I prefer new games copying Blank than the old behavior of combining the other lists

Reply:
> hash words is not None for all our games

maybe misunderstand, but that's not true right now. Corruption, super metroid and maybe some other games, have currently None for the hashwords: 
![grafik](https://github.com/randovania/randovania/assets/38186597/34f286c6-b1ee-49b8-9518-2e7043a56c38)


Or do you mean that I should change it so they also copy blanks words list and then remove the `| None` from here?

Reply:
I have lost count of the number of games, great. Probably a good idea to copy then.